sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -27,6 +27,8 @@ select right("abcd", -2), right("abcd", 0), right("abcd", 'a');
 SELECT split('aa1cc2ee3', '[1-9]+');
 SELECT split('aa1cc2ee3', '[1-9]+', 2);
 SELECT split('hello', '');
+SELECT split('hello', '', 1);
+SELECT split('hello', '', 3);
 SELECT split('', '');
```

Comment:
I would also prefer to see:
```
SELECT split('', '', -1);
SELECT split('', '', 0);
SELECT split('', '', 1);
```
here, for more complete testing

Reply:
Thanks, already added.